src/input.cpp

```diff
@@ -1355,6 +1356,34 @@ cata::optional<tripoint> input_context::get_coordinates( const catacurses::windo
 }
 #endif
 
+std::tuple<point, bool> input_context::get_coordinates_inventory( const catacurses::window
+        &capture_win ) const
+{
+    std::tuple<point, bool> tuple;
+    if( !coordinate_input_received ) {
+        return std::make_tuple( point(), false );
+    }
+
+    const window_dimensions dim = get_window_dimensions( capture_win );
+
+    const int &fw = dim.scaled_font_size.x;
+    const int &fh = dim.scaled_font_size.y;
+    const point &win_min = dim.window_pos_pixel;
+    const point &win_size = dim.window_size_pixel;
+    const point win_max = win_min + win_size;
+
+    const rectangle win_bounds( win_min, win_max );
+
+    const point screen_pos = coordinate - win_min;
+    const point selected( screen_pos.x / fw, screen_pos.y / fh );
+
+    if( !win_bounds.contains_inclusive( coordinate ) ) {
```

Comment:
```suggestion
    if( !win_bounds.contains_half_open( coordinate ) ) {
```

Reply:
Actually that kind of one pixel thing doesn't matter. As you can see, input_context::get_coordinates in sdltiles.cpp is using contains_inclusive, while input_context::get_coordinates in input.cpp which is next to this method is using contains_half_open.
Besides, I also don't understand whats the different.